domain/src/main/java/org/oppia/android/domain/oppialogger/logscheduler/MetricLogSchedulingWorker.kt

```diff
@@ -8,11 +8,15 @@ import com.google.common.util.concurrent.SettableFuture
 import kotlinx.coroutines.CoroutineDispatcher
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.async
+import org.oppia.android.domain.oppialogger.analytics.ActivityLifecycleObserver
+import org.oppia.android.domain.oppialogger.analytics.PerformanceMetricsLogger
 import org.oppia.android.domain.util.getStringFromData
 import org.oppia.android.util.logging.ConsoleLogger
 import org.oppia.android.util.threading.BackgroundDispatcher
 import javax.inject.Inject
 
+private const val BACKGROUND_WORKER_SCREEN = "background_worker_screen"
```

Comment:
Could you dedicate one of the CurrentScreen enum constants for this case to avoid needing to use strings everywhere? Stronger types are preferred where possible.

Reply:
done.